test/proxy/write_to_proxies.json

```diff
@@ -0,0 +1,74 @@
+[
+    {{ range $idx, $offset := int_range 0 (datastore "max_entries") }}
+        {{ if gt $idx 0 }},{{ end }}
+        {
+            "name": "Add some JSON requests to proxy 'test'",
+            "request": {
+                "server_url": "{{ datastore "req_base_url" }}",
+                "endpoint": "proxywrite/test",
+                "method": "POST",
+                "query_params": {
+                    "some": "value"
+                },
+                "header": {
+                    "X-Some": "x-header"
+                },
+                "body": {
+                    "some": [
+                        "body", "here"
+                    ],
+                    "index": "idx_{{ $offset }}"
+                }
+            },
+            "response": {
+                "statuscode": 200,
+                "body": {
+                    "offset": {{ $offset }}
+                }
+            }
+        }
+    {{ end }}
+    ,{
+        "name": "post image to proxy 'test'",
+        "request": {
+            "server_url": "{{ datastore "req_base_url" }}",
+            "endpoint": "proxywrite/test",
+            "method": "POST",
+            "body_file": "@../_res/assets/camera.jpg",
+            "body_type": "file"
+        },
+        "response": {
+            "statuscode": 200,
+            "body": {
+                "offset": {{ datastore "max_entries" }}
+            }
+        },
+        "store_response_qjson": {
+            "test_image_offset": "body.offset"
+        }
+    }
+    {{ range $idx, $offset := int_range 0 (datastore "max_entries") }}
+        ,{
+            "name": "Add some JSON requests to proxy 'test2'",
+            "request": {
+                "server_url": "{{ datastore "req_base_url" }}",
+                "endpoint": "proxywrite/test2",
+                "method": "GET",
+                "query_params": {
+                    "some": "value",
+                    "my": "data_{{ $offset }}",
+                    "is": "here"
+                },
```

Comment:
This is not tested in the "read_from_proxies". Check the "RawQuery" there.  Since the order can differ, you could at least test for "me=data_...". i think we have a :control with regexp

Reply:
It is, check 2 reviews above:
```"X-Apitest-Proxy-Request-Query": ["is=here&my=data&some=value"], // The request query string only```